folly/futures/test/RetryingTest.cpp

```diff
@@ -165,7 +165,8 @@ TEST(RetryingTest, large_retries) {
   PCHECK(getrlimit(RLIMIT_AS, &oldMemLimit) == 0);
 
   rlimit newMemLimit;
-  newMemLimit.rlim_cur = std::min(1UL << 30, oldMemLimit.rlim_max);
+  newMemLimit.rlim_cur =
+      std::min(static_cast<rlim_t>(1UL << 30), oldMemLimit.rlim_max);
```

Comment:
Do you have a preference of `static_cast`ing to the proper type to make the first and second operands to min match up? Some people prefer just specifying the template parameter to `std::min` as it is only templated on one type and so that is why the first and second operand must have the same type to get the auto template deduction to work. I don't have a strong opinion on this obviously, but didn't know if Folly had some standard that others follow for this sort of thing. 

Example:
`newMemLimit.rlim_cur =  std::min<rlim_t>(1UL << 30, oldMemLimit.rlim_max);`

Reply:
This is fine - thanks for checking.

Note that this forces the two arguments to have the same type, rather than allowing both arguments to be cast implicitly to a third type.

Reply:
Thanks for the response. Very fair point on the convertibility.